### DISCUSSION TITLE
No difference between getFullUrl() and getUrl() in v10 with default configuration

### DISCUSSION THREAD

**Discussion by bistory** [spatie/laravel-medialibrary#3222] [Q&A]
Hi,
I just installed this library to test it in my workflow. It works great but when I try to get the relative URL with getUrl(), without the domain name, it returns the full URL like if I called getFullUrl().

I use the default config (not overwritten) and I just call `$ad->getFirstMedia()->getUrl()` in a template.

Please ask if you need more information !
Thanks !

**Comment by patinthehat** [comment]
Hi @bistory, could you provide a failing unit test to demonstrate that this is a bug with the library itself? Thanks!

**Comment by LowellTebrero** [comment]
Help, why is that, I didn't see anything from my `<img src="{{ $post->getFirstMediaUrl('images') }}" alt=""/>` , when I inspect it gets empty `<img src alt> == $0`. I just followed the docs.

**Comment by nicscap** [comment]
Same problem @alexmercertaken .

Can't resolve, over 1 day with this problem... incredible... For my nothing work getMedia, getUrl or other functions.

**Comment by freekmurze** [comment]
I've moved this to a discussion as I'm convinced this problem is not caused by our package. We have test surrounding this functionality and this core functionality seems to be work for the vast majority of users of this package.

If you are having problems with this, it's most likely an installation issue with the package or something in your project.

If you are convinced the package code is wrong or can be improved, do PR a failing test to our testsuite that shows something is off.
